configs/tidyverse.json

```diff
@@ -19,14 +18,6 @@
     ".dont-index"
   ],
   "sitemap_urls": [
-    "https://dplyr.tidyverse.orgsitemap.xml"
-  ],
-  "custom_settings": {
-    "separatorsToIndex": "_"
-  },
-  "scrap_start_urls": false,
-  "conversation_id": [
```

Comment:
Please do not remove this parameter

Reply:
Restored in https://github.com/algolia/docsearch-configs/pull/738/commits/5784aefb0550df4e3c4be5aa85afb68db3df09a2